### ISSUE TITLE
`normalized-lists` is missing on master, and 75/76 branches

### ISSUE THREAD

**Issue by garvankeeley** [mozilla-services/shavar-prod-lists#174]
iOS requires a fingerprinting list in normalized-lists. For instance it exists here on 72 https://github.com/mozilla-services/shavar-prod-lists/tree/72.0

**Comment by garvankeeley** [comment]
@skim1102 are you able to address this so iOS can go forward with https://github.com/mozilla-mobile/firefox-ios/issues/5968 ? Thanks!

**Comment by skim1102** [comment]
Yes, I can get this done by tomorrow. However, this should not block the changes for https://github.com/mozilla-mobile/firefox-ios/issues/5968 from happening since the fingerprinting blocklist is generated from us and as such will not be needed in removing `google-mapping.json` from generating the tracking protection lists within iOS.

Also, on a separate note, I wanted to discuss when the fingerprinting list should be available by for new iOS releases while waiting for automated updates to shavar-prod-lists for the aforementioned fingerprinting file.

**Comment by garvankeeley** [comment]
So iOS should stay on the 72 branch since that is the only valid branch for iOS?
